github-dark.css

```diff
@@ -2512,6 +2512,59 @@
   html.octotree:not(.octotree-show) .zh-board-expanded .zh-board-menu .tabnav-left {
     margin-left: 20px;
   }
+
+  /* Awesome autocomplete styling */
+  .awesome-autocomplete .tt-dropdown-menu {
+    border-color: #343434 !important;
+    background-color: #181818 !important;
+  }
+  .awesome-autocomplete .aa-query strong,
+  .awesome-autocomplete .aa-query-default em,
+  .awesome-autocomplete .tt-suggestion .octicon {
+    color: /*[[base-color]]*/ #4183c4 !important;
+  }
+  .awesome-autocomplete .aa-category {
+    background-color: #1d1d1d !important;
+    border-top: 1px solid #343434 !important;
+    border-bottom: 1px solid #343434 !important;
+    color: #a0a0Aa !important;
+  }
+  .awesome-autocomplete .tt-suggestion {
+    border-color: #343434 !important;
+  }
+  .awesome-autocomplete .tt-suggestion:hover,
+  .awesome-autocomplete .tt-suggestion.tt-cursor {
+    background: #333 !important;
+  }
+  .awesome-autocomplete .tt-suggestion:hover .aa-name {
+    text-decoration: none !important;
+  }
+  .awesome-autocomplete .tt-suggestion em {
+    background-color: /*[[base-color]]*/ #4183c4 !important;
+    color: #fff !important;
+  }
+  .awesome-autocomplete .aa-thumbnail {
+    background-color: #181818 !important;
+  }
+  .awesome-autocomplete .aa-query,
+  .awesome-autocomplete .aa-branding,
+  .awesome-autocomplete .aa-infos,
+  .awesome-autocomplete .aa-login,
+  .awesome-autocomplete .aa-company,
+  .awesome-autocomplete .aa-company i,
+  .awesome-autocomplete .aa-description,
+  .awesome-autocomplete .aa-issue-body {
+    color: #c0c0c0 !important;
+  }
+  .awesome-autocomplete .aa-thumbnail ,
+  .awesome-autocomplete .aa-thumbnail {
+    background-color: #181818 !important;
+  }
```

Comment:
Is this the user avatar background? If so, I'd make it slightly brighter to it stands out over the background. See the last user here (not sure why it fails to pull his avatar, he has one):

![](https://cloud.githubusercontent.com/assets/115237/14029632/81252fde-f203-11e5-999a-8a7579a764ea.png)